wp-content/plugins/ig-wp-api-extensions/endpoints/v0/RestApi_ModifiedContent.php

```diff
@@ -241,6 +246,7 @@ protected function map_post_to_foreign_language_id($post, $language_code) {
 		return $id;
 	}
 
+	/* Probably unused function. available_language_urls uses the function in line 14 */
```

Comment:
I agree. Let's remove the function. Unless @maxammann you had some other intentions here?